library/Solarium/QueryType/Select/Query/Component/DisMax.php

```diff
@@ -259,6 +259,13 @@ public function getTie()
         return $this->getOption('tie');
     }
 
+    /** 
+     * Boost queries
+     *
+     * An array with boostqueries
+     */
+    protected $_boostQueries = array();
```

Comment:
Solarium uses the Symfony coding standard, which doesn't use ZF-style underscores for protected vars. Could you update that?